src/set_theory/zfc/ordinal.lean

```diff
@@ -16,33 +16,38 @@ Further development can be found on the branch `von_neumann_v2`.
 
 ## Definitions
 
-- `Set.is_transitive` means that every element of a set is a subset.
+- `Set.is_transitive`: every element of a set is a subset.
+- `Set.is_ordinal`: a set is hereditarily transitive.
 
 ## Todo
```

Comment:
I guess the first thing to do is an order_iso between all vN ordinals (ordered by `(⊆)`) and [ordinal](https://leanprover-community.github.io/mathlib_docs/set_theory/ordinal/basic.html#ordinal). Seems you haven't proven ` (⊂)` is the same as `(∈)` between two vN ordinals?